daisy/step_create_images.go

```diff
@@ -143,17 +147,24 @@ func (c *CreateImages) run(ctx context.Context, s *Step) error {
 		wg.Add(1)
 		go func(ci *CreateImage) {
 			defer wg.Done()
-
-			project := strOr(ci.Project, w.Project)
```

Comment:
I assume this was some leftover thing before populate(). If so, thanks for the cleanup.